src/test/java/org/elasticsearch/test/ExternalNode.java

```diff
@@ -141,6 +140,7 @@ synchronized void startInternal(Client client, Settings settings, String nodeNam
             if (waitForNode(client, nodeName)) {
                 nodeInfo = nodeInfo(client, nodeName);
                 assert nodeInfo != null;
+                logger.debug("external node {} found, version [{}], build {}", nodeInfo.getNode(), nodeInfo.getVersion(), nodeInfo.getBuild());
```

Comment:
that what I was originally thinking, but the start node message is debug. maybe both to info?